src/Xenial.Tasty.Cli/Commands/StudioCommand.cs

```diff
@@ -34,23 +56,44 @@ public static Task<int> Studio(CancellationToken cancellationToken)
                             AllowsMultipleSelection = false
                         };
                         Application.Run(dialog);
-                        var filePaths = dialog.FilePaths;
+                        var filePath = dialog.FilePath;
+                        if(filePath != null)
+                        {
+                            var path = filePath.ToString();
+                            if(!string.IsNullOrEmpty(path) &&  System.IO.File.Exists(path))
```

Comment:
System.IO.File.Exists checks for null or empty and returns false.

Reply:
This is true, but MS did not annotate the `File.Exists` with the `NotNull` annotation. So a usage of path fails later.